crates/router/src/core/admin.rs

```diff
@@ -1383,7 +1383,7 @@ impl ConnectorAuthTypeAndMetadataValidation<'_> {
             | api_enums::Connector::DummyConnector5
             | api_enums::Connector::DummyConnector6
             | api_enums::Connector::DummyConnector7 => {
-                dummyconnector::transformers::DummyConnectorAuthType::try_from(self.auth_type)?;
+                hyperswitch_connectors::connectors::dummyconnector::transformers::DummyConnectorAuthType::try_from(self.auth_type)?;
```

Comment:
Nit:
Use relative import
```suggestion
                dummyconnector::transformers::DummyConnectorAuthType::try_from(self.auth_type)?;
```